packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -6,7 +6,8 @@ export type PreviewOptions = 'Desktop' | 'Mobile' | 'Tablet';
 const panel = 'div.interface-interface-skeleton__header';
 const selectors = {
 	// Block Inserter
-	blockInserterButton: `${ panel } button.edit-post-header-toolbar__inserter-toggle`,
+	// Note the partial class match. This is to support site and post editor. We can't use aria-label because of i18n. :(
+	blockInserterButton: `${ panel } button[class*="header-toolbar__inserter-toggle"]`,
```

Comment:
I'm surprised you had to change this. I thought the CSS class would match regardless of mobile/desktop, and i18n.

Reply:
>I thought the CSS class would match regardless of mobile/desktop, and i18n

Oh, the change was to support the site editor, not necessarily across viewports! The i18n comment is just explaining why we can't use `aria-label` (which is the same across all editors)

Reply:
Ah okay, got it.